portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -52,7 +52,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     storeComment(commenter, content);
 
     // Redirects back to the HTML page.
-    response.sendRedirect("/index.html");
+    response.sendRedirect("/comments.html");
```

Comment:
Look like these redirection path could be defined as constant above?

Reply:
Fixed